tensilelite/Tensile/KernelWriter.py

```diff
@@ -3943,18 +3943,18 @@ def readWriteVectors(mat, vw, kernel):
     self.states.b.numVgprLocalWriteAddr = 0 if kernel["LocalWriteUseSgprB"] else 1 * self.states.rpla
 
     if self.states.archCaps["HasLDSGT64K"] and not kernel["1LDSBuffer"] and not kernel["LocalWriteUseSgprA"] :
-      if kernel["LdsOffsetA_Blk"]>=131072:
+      if kernel["LdsOffsetA_Blk"]>=131072 or kernel["LdsNumElementsAlignedA"]>=131072:      
         self.states.a.numVgprLocalReadAddr =3* self.states.rpla
```

Comment:
Not directly related to this change, but numVgprLocalReadAddr should not be affected by LocalWriteUseSgprA.
I would suggest to use 
        self.states.a.numVgprLocalReadAddr *=3
here and remove not kernel["LocalWriteUseSgprA"] at line 3945.

Reply:
LocalWriteUseSgprA is already 0 in case of kernel["LocalWriteUseSgprA"].
So, we can simply use *=3 for both numVgprLocalWriteAddr and numVgprLocalReadAddr without checking LocalWriteUseSgprA again.